libcontainer/cgroups/systemd/v2.go

```diff
@@ -27,14 +27,18 @@ type unifiedManager struct {
 	path     string
 	rootless bool
 	dbus     *dbusConnManager
+	// host UID corresponding to UID 0 in container namespace;
+	// or -1 if cgroup should not be chown'd
+	uid int
 }
 
-func NewUnifiedManager(config *configs.Cgroup, path string, rootless bool) cgroups.Manager {
+func NewUnifiedManager(config *configs.Cgroup, path string, uid int, rootless bool) cgroups.Manager {
```

Comment:
I'm afraid we can't change this function signature, as other software (kubernetes, cri-o, possibly cAdvisor etc.) depend on it (unfortunately).

Maybe you can add a separate method to set the uid?

Reply:
or a `WithUID()` modifier method.

Reply:
Thanks, I will do as you suggest.